pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -71,7 +71,7 @@ def __init__(
         self.debug = debug
         self.tpu_local_core_rank = 0
         self.tpu_global_core_rank = 0
```

Comment:
We are removing this as well, right?

Reply:
I've been working on it over here #11163